Tasks/VsTest/vstest.ts

```diff
@@ -1533,3 +1490,17 @@ function responseContainsNoTests(filePath: string): Q.Promise<boolean> {
         }
     });
 }
+
+function isNullOrWhitespace(input) {
+    if (typeof input === 'undefined' || input == null) {
+        return true;
+    }
+    return input.replace(/\s/g, '').length < 1;
+}
+
+function getTaskMatchOption(): any {
```

Comment:
we should drive consistency across tasks: `{ dot: true, nobrace: true, nocase: process.platform == 'win32' }` or leave empty and the lib will apply those defaults